test-e2e.sh

```diff
@@ -104,9 +104,9 @@ set -o xtrace
 
 # Run end-to-end integration testing.
 # First, GRPC.
-./examples/hellogrpc/e2e-test.sh $E2E_NAMESPACE cc java go py
+./examples/hellogrpc/e2e-test.sh remote $E2E_NAMESPACE cc java go py
 # Second, HTTP.
-./examples/hellohttp/e2e-test.sh $E2E_NAMESPACE java go py nodejs
+./examples/hellohttp/e2e-test.sh remote $E2E_NAMESPACE java go py nodejs
 # Third, TODO Controller.
 # chrislovecnm - disabled till this is less flakey
 # ./examples/todocontroller/e2e-test.sh $E2E_NAMESPACE py
```

Comment:
also modify here to add remote arg

Reply:
This test now fails remotely for me all the time. And I wasn't sure if I can make it work locally without using gcloud once it's fixed (which is why I left it as is).

I now added the remote arg, modified examples/todocontroller/e2e-test.sh to only work with the remote arg and added a TODO to try make it run locally when it's fixed for running remotely.